### DISCUSSION TITLE
blank screen after logging in (Win 11 23H2)

### DISCUSSION THREAD

**Discussion by nazzy121** [valinet/ExplorerPatcher#3169] [Q&A]
Anyone getting a blank screen after logging in?  My PC was fine until after installing April 2024 cumulative updates.  Details below.   

OS: Windows 11 23H2
Latest Cumulative Update: 2024-04
ExplorerPatcher version: 22621.3296.64.1

On a domain-joined PC after user logs into computer, the screen is completely blank, probably due to explorer.exe not starting correctly. Workaround is to open task manager (via hotkey) and run new task "explorer.exe". I don't know if being domain-joined or not has any effect, but it's the only PC I have to test with.

This started happening today after ExplorerPatcher updated itself and/or downloaded latest symbols after I installed April 2024 Cumulative Updates on Win11 23H2.

**Comment by pyrates999** [comment]
Update to a pre-release, it's been fixed there: https://github.com/valinet/ExplorerPatcher/issues/2862

**Reply by nazzy121** [reply]
Updated to 3527.65.1 but still the same blank screen after log in.

**Reply by pyrates999** [reply]
Does it happen every time you login?  Or just the first time?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by nazzy121** [comment]
It was happening every time I logged in last week.  Tried rebooting a dozen or so times.  

However today the blank screen problem seems to have gone away, at least for now. I rebooted 6 times.  

Interestingly, I had 3527.65.1 already installed (as per my previous post last week), but today the auto-update function sent a notification that this same update was available.  I clicked on "update" to see if it would do anything and it proceeded to download/install the same version I already had. 

If blank screen problem returns in the future I will post here again.  Thanks for your help so far.  

Windows 11 23H2 22631.3447
Apps running that change/affect Windows UI: 
1) 7+ Taskbar Tweaker
2) Rainmeter

**Comment by pyrates999** [comment]
If you have EP set to check for pre-releases, it would show there was an update when a new pre-release was uploaded.

**Comment by Amrsatrio** [comment]
Was that the very first 63.1 that said 50/50 chance? If so, that build is flawed under 22631. The latest 65.1 that doesn't say 50/50 doesn't have the black screen issue. I will make a new release soon.

**Reply by nazzy121** [reply]
Unsure. My reply above, "Updated to 3527.65.1 but still the same blank screen after log in."  was posted 5-10 mins after I installed the pre-release from here.  At any rate, the problem went away so hopefully it was fixed for good.

**Comment by pyrates999** [comment]
There is now a 22621.3527.65.2 release.  Please update to it.
